README.md

```diff
@@ -13,39 +13,119 @@ Artifacts in support of ODH Edge use cases that show integration with Red Hat Ad
-1. Build the model from the notebook using Data Science Pipelines
-1. Push the model to the image registry accessible by the near edge cluster(s)
-1. Update the GitOps config for the near edge cluster
-
-### Pipelines setup
-
-See [pipelines/README.md](pipelines/README.md)
+Working and deploying your own models might require bigger changes
+to the definition and configuration of the pipelines and ACM setup below,
+so you might want to start with the pre-built models first.
+
+### Building container image with model and serving runtime
+
+In [pipelines/README.md](pipelines/README.md) we show how to take the trained models,
+store them in a S3 bucket,
+build container image(s) with the model and serving runtime using OpenShift Pipelines,
+push the container image(s) to an image registry accessible by the near edge cluster(s),
+and update a clone of this repository with a pull request,
+configuring `acm/odh-edge/apps/*/kustomization.yaml` with location and digest (SHA-256) of the built images.
+
+You can skip this step if you do not wish to rebuild the container images.
+If you use the default configuration as shown in this git repository,
+you will use already built container images from the https://quay.io/organization/rhoai-edge repositories.
+
+### Serving to near edge remote clusters
+
+We assume that you have an admin access to an OpenShift cluster where you can install and configure
+the Red Hat Advanced Cluster Management operator and which will serve as the central ACM hub,
+and other OpenShift cluster(s) with admin privileges that you will import to the ACM hub cluster
+and that will play the role of near edge location(s), managed from the central ACM hub.
+
+#### ACM setup
+
+The following setup steps need to be done as an admin user with `cluster-admins` privileges.
+We will describe how to do them using OpenShift Console but there may be other ways to achieve the same results.
+
+1. On the ACM hub cluster, install the Advanced Cluster Management for Kubernetes operator.
+   In OpenShift Console of the cluster, find and run the installation of the operator
+   in Menu > Operators > OperatorHub.
+1. Create the MultiClusterHub.
+   In OpenShift Console in Menu > Operators > Installed Operators > Advanced Cluster Management for Kubernetes >
+   the MultiClusterHub tab, use the "Create MultiClusterHub" button to start the installation.
+1. Refresh the Web OpenShift Console.
+1. Create a cluster set for your near edge clusters.
+   In OpenShift Console it can be done in All Clusters menu > Infrastructure > Clusters > Cluster sets tab.
+   This repository assumes name `poc-near-edge` in `clusterSets`
+   in [acm/registration/near-edge/base/near-edge.yaml](acm/registration/near-edge/base/near-edge.yaml)
+   so update that YAML file if you use different name.
+1. Import the near edge cluster(s).
+   In OpenShift Console it can be done in All Clusters menu > Infrastructure > Clusters > Cluster list tab.
```

Comment:
```suggestion
   In the OpenShift Console this can be done in All Clusters menu > Infrastructure > Clusters > Cluster list tab.
```

Reply:
Applied in 49aadf7f7022a32884be3d91d13200f348ad82ab.